lib/beyond_api.rb

```diff
@@ -46,6 +46,15 @@ def initialize
       @log_bodies = false
 
       @all_pagination_size = 200
+
+      @retry_options = {
+        max: 5,
+        interval: 0.05,
+        interval_randomness: 0.5,
+        backoff_factor: 2,
+        retry_statuses: [409],
+        exceptions: [Faraday::TimeoutError, Faraday::ConnectionFailed]
+      }
```

Comment:
To avoid extra logic I wrote in the docs the option to overwrite single values, or to send the entire config:
```ruby
  # ==> Retry configuration
  # Configure the retry options for requests. Default is:
  # config.retry_options = {
  #   max: 5,
  #   interval: 0.05,
  #   interval_randomness: 0.5,
  #   backoff_factor: 2,
  #   retry_statuses: [409],
  #   exceptions: [Faraday::TimeoutError, Faraday::ConnectionFailed]
  # }
  # Or you can set single retry options like this:
  # config.retry_options[:max] = 5
  # config.retry_options[:interval] = 1
  ```

Also if the config is missing it will take the `faraday-retry` deafults. In addition to this, you can disable the retries with `config.retry_options[:max] = 0`